wagtail/admin/views/pages/listing.py

```diff
@@ -71,10 +86,26 @@ class PageFilterSet(WagtailFilterSet):
         queryset=Site.objects.all(),
         widget=CheckboxSelectMultiple,
     )
+    has_child_pages = HasChildPagesFilter(
+        label=_("Has child pages"),
+        widget=Select(
```

Comment:
Any reason why not use a `RadioSelect` instead? Seems to be more intuitive that way IMO, one less click to apply the filter.